lava_test_plans/testplans/mmtests-benchmarking/mmtests-sysbench-cpu.yaml

```diff
@@ -0,0 +1,5 @@
+{% extends "testcases/master/template-mmtests.jinja2" %}
+
+{% set testname = 'sysbenchcpu' %}
+{% set configfile = 'configs/config-scheduler-sysbench-cpu' %}
+{% set test_timeout = 45 %}
```

Comment:
In the testplans/mmtest-benchmarking directory, files in there should be symlinks to files inside testcases/mmtests-sysbench-cpu.yaml.

However, with only one file I'm not convinced its needed.

Reply:
Removed as outdated approach